mercadopago/px-ios
# (Fixed)[PXN-4286] Fallback x-platform field in checkout init to MP

## What has changed?

Fixed _x-platform_ fallback from ML to MP, as this is the way our backend expects it. 

This fixes issues with ODR assets and deeplinks that were being shown incorrectly when the API call to [api.mercadopago.com/platforms/ios/versions/xxx](http://api.mercadopago.com/platforms/ios/versions/xxx) fails (e.g., when creating a MDS)

## How to test:

Check x-platform field sent when initializing the checkout. 
An example of this fix is in MDS named _IFPE PX 2.0_. vs _IFPE PX_
